examples/Cookbook.ipynb

```diff
@@ -654,6 +789,70 @@
     "print(transaction_converter.transaction_to_dictionary(relayed_tx))"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Relayed V3\n",
+    "\n",
+    "This is the latest iteration of relayed transactions and we highly recommend using this version over the previous ones."
```

Comment:
Add a link to the official docs. And maybe rephrase to say "we recommend using this version ... as soon as it becomes available on the Network" (or something similar).

Reply:
changed the message and will add a link to the docs when that is available.